feat: Added new method: check for new unlisted tags (#3437)
coreruleset/coreruleset: util/crs-rules-check/README.md

    * the variable is an operator argument, eg `SecRule ARGS "@rx %{TX.foo}"...`
    * the variable is a right hand side operand in a `setvar` action, eg `setvar:tx.bar=%{tx.foo}`
    * the variable is in an expansion, e.g., as part of the value of a `msg` action: `msg:'Current value of variable: %{tx.foo}`

> * **Check new tags at rules** - cannot be added to a new tag rule unless it is in the TAGS_USED file

Reviewer: ```suggestion
* **Check rule tags** - only tags listed in `TAGS_USED` may be used as tags in rules
```

>     * before you want to add a new tag to a rule, you **must** add it to TAGS_USED file

Reviewer: ```suggestion
    * to use a new tag on a rule, it **must** first be registered in the TAGS_USED file
```
